docs/replanning_tutorial.rst

```diff
@@ -0,0 +1,67 @@
+Relinking
```

Comment:
can you add these as sub pages to 
`tutorial.rst`? right now this page and the other one won't show up